### DISCUSSION TITLE
How to tailor the results to be domain user friendly? (newbie q.)

### DISCUSSION THREAD

**Discussion by paulio** [reqnroll/Reqnroll#191] [Q&A]
I'm currently running with MSTest + NUnit and the report shows that the feature passes, and the step information is there if you dig into the standard output. I'd really like to be able to create a report where the steps and their results are more easily displayed for a domain user to consume. I don't wish to 'lead the witness', but what would be a good direction for me? E.g. is there an output converter, a log runner, a result parser/converter, etc.

**Comment by gasparnagy** [comment]
Yes, this is part of our plans, but there is no built-in solution for it yet. See related discussion: https://github.com/orgs/reqnroll/discussions/37

As a simple solution, you can use the TRX test result with some basic HTML conversion. I have used https://github.com/stevencohn/trxer for that which produces an OK result.
